aiida_mlip/calculations/base.py

```diff
@@ -217,13 +217,17 @@ def prepare_for_submission(
             cmd_line["arch"] = architecture
 
         model_path = None
-        if "model" in self.inputs:
+        if "model" in self.inputs and self.inputs.model is not None:
             model_path = self.inputs.model.filepath
+        # Raising error if model is None
+        elif "model" in self.inputs and self.inputs.model is None:
+            raise ValueError("Model cannot be None")
```

Comment:
```suggestion
        if "model" in self.inputs:
            # Raise error if model is None
            if self.inputs.model is None:
                raise ValueError("Model cannot be None")
            model_path = self.inputs.model.filepath
```

Stylistic preference, but it's clearer to me if we group by what's in `self.inputs` on its own